Index: equation_trajectoire.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from math import*\r\n\r\n#trajectory equation of arrow and portal, v0 is initial speed, theta is the initial angle, t is time, g is the gravity\r\n\r\ndef trajectoire(v0,theta,t,g) :\r\n    Xposition = v0 * cos(theta) * t\r\n    Yposition = (-1/2) * ((g*t)^2) * v0 * sin(theta) * t\r\n    coordonnee = []\r\n    coordonnee[0] = Xposition\r\n    coordonnee[1] = Yposition\r\n    return coordonnee\r\n\r\ndef power(dt) : #dt is the time that us\r\n    Pmin = 50 #minimal power\r\n    CP = 2 #coefficient of proportionnality of the power from the arrow\r\n    P = CP * dt + Pmin\r\n    return P\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/equation_trajectoire.py b/equation_trajectoire.py
--- a/equation_trajectoire.py	(revision 08504784816477a58565c19555b52935d96bda56)
+++ b/equation_trajectoire.py	(date 1739800711614)
@@ -14,4 +14,4 @@
     Pmin = 50 #minimal power
     CP = 2 #coefficient of proportionnality of the power from the arrow
     P = CP * dt + Pmin
-    return P
+    return P
\ No newline at end of file
